### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by kschulst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+
+> TLDR: Maskinporten Guardian exchanges keycloak access tokens for Maskinporten access tokens.
+
+The Guardian manages sensitive information such as certificates and secrets. It keeps track of who can retrieve access
+tokens on behalf of different _Maskinporten clients_. The Guardian also handles audit logging, making it easy to see
+_who_ requested access to _what_ API _when_.
+
+## What's Maskinporten?
+
+Maskinporten is a national, common solution providing secure authentication and access control that enables data
```
Comment (mmj-ssb) [inline README.md:16]: Comma after 'common solution' I think.

Comment (kschulst) [reply README.md:16]: I beg to differ ;)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+exchange between different Norwegian data providers. A [growing number](https://samarbeid.digdir.no/maskinporten/maskinporten/995)
+of APIs are being protected by Maskinporten.
+
+Access to Maskinporten is administered via _Samhandlingsportalen_. Organizations (e.g., SSB) that need to consume an
```
[the remark this answers is not in the document]
Comment (mmj-ssb) [reply README.md:20]: No idea..

File: README.md [hunk 3]
```diff
@@ -1,6 +1,58 @@
...
+So, you want to become a trusted party? You will need to:
+
+1. Get hold of a maskinporten client ID associated with the API(s) that you want to consume. Administrators of
+   _Samhandlingsportalen_ can help you with this. If in doubt, post a question to Team Argus on the
+   #hjelp_dapla slack channel.
+
+2. You will need information about the API you want consume - including the required Maskinporten _data scopes_. This
+   is typically something you will need to get from the API provider.
+
+3. The default and preferred way to access Maskinporten Guardian is via so called _Service Account Users_. These users
```
[the remark this answers is not in the document]
Comment (kschulst) [reply README.md]: Exactly how do you think a comma would be placed here?

**Pull request merged**